packages/api/src/controllers/user.ts

```diff
@@ -60,6 +62,39 @@ async function findUserByEmail(email: string, useReplica = true) {
   return users[0];
 }
 
+const frontendUrl = (
```

Comment:
Nice! Maybe `buttonUrl = (` as it's always used for the `buttonUrl` field?

Reply:
I actually had `emailUrl` since they're always included in emails. I don't think that makes a lot of sense tho since you would then have to look at where this is used to make sense out of it, instead of it making sense on its own. Then I thought that the most appropriate name would be this, since what it really does is create a URL pointing to a frontend page.